examples/README.md

```diff
@@ -35,13 +35,10 @@ This configuration will install application for windows worker node only.
 
 All windows images are available in a separate `quay.io` repository: `quay.io/signalfx/splunk-otel-collector-windows`.
 
-Windows support covers traces and metrics collection only, no logs will be collected. As of now, we don’t have fluentd-hec image for the windows worker node.
```

Comment:
Please mention that logs collection on Windows is only available using OTel native logs collection, not fluentd.

Reply:
Updated documentation section